docs/template-config.md

```diff
@@ -116,8 +116,10 @@ Possible values: Any `string`
 - `$repository`: will be replaced with the repository's name in which the issue was found.
 - `$age`: will be replaced with the time passed since the creation of the issue.
 - `$link`: will be replaced with an HTTP url leading to the issue.
-- `$assignee-logins`: will be replaced with a comma separated list of GitHub login names.
-- `$assignee-login-links`: will be replaced with a comma separated list of GitHub login names linking to the GitHub profile pages.
+- `$assignee-logins`: will be replaced with a comma separated list of GitHub login names from assigned reviewers on an Issue or PR.
+- `$assignee-login-links`: will be replaced with a comma separated list of GitHub login names linking to the GitHub profile pages from assigned reviewers on an Issue or PR.
+- `$reviewer-logins`: will be replaced with a comma separated list of GitHub login names from requested reviewers on a PR.
```

Comment:
I believe you meant to add  the new placeholder to the `line-prs` section (one above) since issues don't have reviewers.